From df54ef5ebafbc8817c21ce4c8a2089e638ce7f90 Mon Sep 17 00:00:00 2001
From: axxx007xxxz <mattiabrsrn7@gmail.com>
Date: Sat, 17 Feb 2018 23:28:15 +0100
Subject: [PATCH 3/3] libcutils: fs_config: Update permissions for
 /system/build.prop

Allow "group" and "others" to read it.

It's needed on some device in order to get camera proprietes loaded properly.

Change-Id: If2bdd1c56a6d95b2a455f1d26fc7bd8a59614477
---
 libcutils/fs_config.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libcutils/fs_config.cpp b/libcutils/fs_config.cpp
index cb15b57b8..4859ca93b 100644
--- a/libcutils/fs_config.cpp
+++ b/libcutils/fs_config.cpp
@@ -149,7 +149,7 @@ static const struct fs_path_config android_files[] = {
     { 00750, AID_ROOT,      AID_ROOT,      0, "system/bin/install-recovery.sh" },
     { 00700, AID_ROOT,      AID_ROOT,      0, "system/bin/secilc" },
     { 00750, AID_ROOT,      AID_ROOT,      0, "system/bin/uncrypt" },
-    { 00600, AID_ROOT,      AID_ROOT,      0, "system/build.prop" },
+    { 00644, AID_ROOT,      AID_ROOT,      0, "system/build.prop" },
     { 00444, AID_ROOT,      AID_ROOT,      0, sys_conf_dir + 1 },
     { 00444, AID_ROOT,      AID_ROOT,      0, sys_conf_file + 1 },
     { 00440, AID_ROOT,      AID_SHELL,     0, "system/etc/init.goldfish.rc" },
-- 
2.17.1

